BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/html_file/views/_html.html.erb

```diff
@@ -0,0 +1,41 @@
+
+<%
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2023
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+%>
+
+
+
+<%= link_to h("Open in a separate tab"), url_for(:controller  => :userfiles, :id  => @userfile.id, :action  => :display, :viewer => :html, :content_viewer => "on", :file_name => @userfile.name), :target => "_blank" if params[:action] == "show" %>
```

Comment:
Is this link just opening the same current page in a new tab, or something different from the current page?

In which situations would "params[:action]" not be "show" ?

Reply:
That link opens other userfile in iframe, without the meta info. 

The condition lets me hide this link, assuming there is no need to show.

The action in that case is `display`

That said can delete this condition or any line if you are not comfortable.

Reply:
OK